data/json/npcs/exodii/exodii_weaponmaster_talk.json

```diff
@@ -2,17 +2,17 @@
   {
     "id": "TALK_WEAPONMASTER_INTRO_1",
     "type": "talk_topic",
-    "//": "mute by choise, and deaf most of the time (don't like to turn on the hearing cbm unless for job)",
-    "dynamic_line": "&What you see looks like the back of a mechanical spider, with at least a dozen crude but polished metal arms going from the enormously wide spine and shoulders.  Spine of a creature, meanwhile, sits in an awkward-looking chair, doing something on a wide, heavily lighted workbench.  Creature seems to be busy with something.",
+    "//": "mute by choice, and deaf most of the time (don't like to turn on the hearing cbm unless for job)",
+    "dynamic_line": "&What you see looks like the back of a mechanical spider, with at least a dozen crude but polished metal arms coming from the enormously wide spine and shoulders.  Spine of a creature, meanwhile, sits in an awkward-looking chair, doing something on a wide, heavily lighted workbench.  Creature seems to be busy with something.",
```

Comment:
Spine of a creature, meanwhile, -> The spine of the creature, meanwhile,
Creature seems to -> The creature seems to